# 获取参考文献



In [5]:
* 路径设定

global path_PDF "D:\Github\lianxhta\Python\refs"

cd $path_PDF



D:\Github\lianxhta\Python\refs


In [2]:
local doi 10.1016/j.jeconom.2025.106009
getiref `doi' , m dis

  
Sun, Z., & Wüthrich, K. (2025). Pairwise valid instruments. Journal of Economet
> rics, 250, 106009.
    Link    PDF    Google    

Sun, Z., & Wüthrich, K. (2025). Pairwise valid instruments. Journal of Economet
> rics, 250, 106009. [Link](https://doi.org/10.1016/j.jeconom.2025.106009), [PD
> F](http://sci-hub.ren/10.1016/j.jeconom.2025.106009), [Google](<https://schol
> ar.google.com/scholar?q=Pairwise valid instruments>).
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^
 
Sun_2025_Pairwise_valid_instruments


Sun, Z., & Wüthrich, K. (2025). Pairwise valid instruments. Journal of Econometrics, 250, 106009. [Link](https://doi.org/10.1016/j.jeconom.2025.106009), [PDF](http://sci-hub.ren/10.1016/j.jeconom.2025.106009), [Google](<https://scholar.google.com/scholar?q=Pairwise valid instruments>).

In [2]:
getiref    2509.11519     , m ar pdf

  
Yao, M., Wang, A., Li, X., & Liu, Z. (2025). Mendelian Randomization Methods fo
> r Causal Inference: Estimands, Identification and Inference (Version 1). arXi
> v.
    Link (rep)    PDF    Google    

Yao, M., Wang, A., Li, X., & Liu, Z. (2025). Mendelian Randomization Methods fo
> r Causal Inference: Estimands, Identification and Inference (Version 1). arXi
> v. [Link](https://doi.org/10.48550/arXiv.2509.11519) (rep), [PDF](https://arx
> iv.org/pdf/2509.11519.pdf), [Google](<https://scholar.google.com/scholar?q=Me
> ndelian Randomization Methods for Causal Inference: Estimands, Identification
>  and Inference (Version 1)>).
        PDF:   dir    view_online     Open
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^


Yao, M., Wang, A., Li, X., & Liu, Z. (2025). Mendelian Randomization Methods for Causal Inference: Estimands, Identification and Inference (Version 1). arXiv. [Link](https://doi.org/10.48550/arXiv.2509.11519) (rep), [PDF](https://arxiv.org/pdf/2509.11519.pdf), [Google](<https://scholar.google.com/scholar?q=Mendelian Randomization Methods for Causal Inference: Estimands, Identification and Inference (Version 1)>).

MR - IV 估计方法

In [4]:
getiref       10.1038/s43586-021-00092-5  , m dis


  
Sanderson, E., Glymour, M. M., Holmes, M. V., Kang, H., Morrison, J., Munafò, M
> . R., Palmer, T., Schooling, C. M., Wallace, C., Zhao, Q., & Davey Smith, G. 
> (2022). Mendelian randomization. Nature Reviews Methods Primers, 2(1).
    Link    PDF    Google    

Sanderson, E., Glymour, M. M., Holmes, M. V., Kang, H., Morrison, J., Munafò, M
> . R., Palmer, T., Schooling, C. M., Wallace, C., Zhao, Q., & Davey Smith, G. 
> (2022). Mendelian randomization. Nature Reviews Methods Primers, 2(1). [Link]
> (https://doi.org/10.1038/s43586-021-00092-5), [PDF](http://sci-hub.ren/10.103
> 8/s43586-021-00092-5), [Google](<https://scholar.google.com/scholar?q=Mendeli
> an randomization>).
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^
 
Sanderson_2022_Mendelian_randomization


In [6]:
getiref 2502.01146  , m ar pdf

  
Du, Y., Wang, X., Guo, N., Yu, Z., Qian, Y., Zhang, K., Hsieh, M.-H., Rebentros
> t, P., & Tao, D. (2025). Quantum Machine Learning: A Hands-on Tutorial for Ma
> chine Learning Practitioners and Researchers (Version 1). arXiv.
    Link (rep)    PDF    Google    

Du, Y., Wang, X., Guo, N., Yu, Z., Qian, Y., Zhang, K., Hsieh, M.-H., Rebentros
> t, P., & Tao, D. (2025). Quantum Machine Learning: A Hands-on Tutorial for Ma
> chine Learning Practitioners and Researchers (Version 1). arXiv. [Link](https
> ://doi.org/10.48550/arXiv.2502.01146) (rep), [PDF](https://arxiv.org/pdf/2502
> .01146.pdf), [Google](<https://scholar.google.com/scholar?q=Quantum Machine L
> earning: A Hands-on Tutorial for Machine Learning Practitioners and Researche
> rs (Version 1)>).
        PDF:   dir    view_online     Open
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^


## AI and Finance

In [8]:
getiref 10.1146/annurev-financial-112923-020503 , m

  
Eisfeldt, A. L., & Schubert, G. (2025). Generative AI and Finance. Annual Revie
> w of Financial Economics.
    Link    PDF    Google    

Eisfeldt, A. L., & Schubert, G. (2025). Generative AI and Finance. Annual Revie
> w of Financial Economics. [Link](https://doi.org/10.1146/annurev-financial-11
> 2923-020503), [PDF](http://sci-hub.ren/10.1146/annurev-financial-112923-02050
> 3), [Google](<https://scholar.google.com/scholar?q=Generative AI and Finance>
> ).
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^


In [12]:
getiref 2402.15585 , m pdf ar

  
Battaglia, L., Christensen, T., Hansen, S., & Sacher, S. (2024). Inference for 
> Regression with Variables Generated by AI or Machine Learning (Version 5). ar
> Xiv.
    Link (rep)    PDF    Google    

Battaglia, L., Christensen, T., Hansen, S., & Sacher, S. (2024). Inference for 
> Regression with Variables Generated by AI or Machine Learning (Version 5). ar
> Xiv. [Link](https://doi.org/10.48550/arXiv.2402.15585) (rep), [PDF](https://a
> rxiv.org/pdf/2402.15585.pdf), [Google](<https://scholar.google.com/scholar?q=
> Inference for Regression with Variables Generated by AI or Machine Learning (
> Version 5)>).
        PDF:   dir    view_online     Open
 
Tips: Text is on clipboard. Press 'Ctrl+V' to paste, ^-^


In [ ]:
getiref  , m pdf

In [ ]:
getiref  , m pdf

In [ ]:
getiref  , m pdf

In [ ]:
getiref  , m pdf

In [ ]:
getiref  , m pdf